library/src/main/java/io/constructor/data/model/search/Path.kt

```diff
@@ -0,0 +1,6 @@
+package io.constructor.data.model.search
+
+import com.squareup.moshi.Json
+import io.constructor.data.model.ProductData
+
+data class Path(@Json(name = "display_name") val displayName: String, val id: Long)
```

Comment:
Not sure what this is

Reply:
its part of the model, guess you guys should know what it is :)

Reply:
I don't think it's used?

Reply:
I don't use it anywhere in SDK, added model to fully parse response

Reply:
Where is it used to parse the response?  Couldn't find a reference to it.